vendor/github.com/operator-framework/api/pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -88,6 +89,10 @@ type CatalogSourceSpec struct {
 	// +optional
 	Secrets []string `json:"secrets,omitempty"`
 
+	// RunAsRoot allows admins to indicate that they wish to run the CatalogSource pod in a privileged
+	// more as root
+	RunAsRoot bool `json:"runAsRoot,omitempty"`
```

Comment:
I assume this is a temp solution for discussion before you make a permanent PR on api repo? Plus, this is considered an API change. Also, do we consider this field to be optional or not? If it ism an `+optional` marker would be nice.

Reply:
Unfortunately the introduction of the new filed isn't a temp solution. It's the only way to get around the problem of "old catalogs cannot run in 4.12 with enforce:restricted enabled by default". I'll update the PR to mark this field as `+optional`

Reply:
Oh right, that's for review. Just opened a PR in the api repo that'll be vendor-ed in https://github.com/operator-framework/api/pull/257